src/components/Chat/ChatInput/ChatInputAttachments.tsx

```diff
@@ -0,0 +1,86 @@
+import {
+  IconExclamationCircle,
+  IconFile,
+  IconReload,
+  IconX,
+} from '@tabler/icons-react';
+import { useCallback } from 'react';
+
+import classNames from 'classnames';
+
+import { FilesActions, FilesSelectors } from '@/src/store/files/files.reducers';
+import { useAppDispatch, useAppSelector } from '@/src/store/hooks';
+
+export const ChatInputAttachments = () => {
+  const dispatch = useAppDispatch();
+  const selectedFiles = useAppSelector(FilesSelectors.selectSelectedFiles);
+
+  const handleUnselectFile = useCallback(
+    (fileId: string) => {
+      return () => dispatch(FilesActions.unselectFiles({ ids: [fileId] }));
+    },
+    [dispatch],
+  );
+
+  const handleRetry = useCallback(
+    (fileId: string) => {
+      return () => dispatch(FilesActions.reuploadFile({ fileId }));
+    },
+    [dispatch],
+  );
+
+  if (!selectedFiles.length) {
+    return null;
+  }
+
+  return (
+    <div className="mb-2.5 grid max-h-[100px] grid-cols-3 gap-1 overflow-auto px-12">
+      {selectedFiles.map((file) => (
+        <div
+          key={file.id}
+          className="flex gap-3 rounded bg-gray-300 p-3 dark:bg-gray-900"
+        >
+          {file.status !== 'FAILED' ? (
+            <IconFile className="shrink-0 text-gray-500" size={18} />
+          ) : (
+            <IconExclamationCircle
+              className="shrink-0 text-red-800 dark:text-red-400"
+              size={18}
+            />
+          )}
+
+          <div className="flex grow justify-between gap-3 overflow-hidden">
+            <div className="flex grow flex-col overflow-hidden">
+              <span
+                className={classNames(
+                  'block max-w-full truncate',
+                  file.status === 'FAILED' && 'text-red-800 dark:text-red-400',
```

Comment:
see recomended usage format [here](https://www.npmjs.com/package/classnames#usage)
```suggestion
                  { 'text-red-800 dark:text-red-400' : file.status === 'FAILED' },
```

Reply:
@IlyaBondar I don't think this is recommended - just possibilities how to use